sqlglot/generator.py

```diff
@@ -2345,6 +2346,20 @@ def ordered_sql(self, expression: exp.Ordered) -> str:
                     f"'{nulls_sort_change.strip()}' translation not supported in window functions"
                 )
                 nulls_sort_change = ""
+            elif (
+                self.NULL_ORDERING_SUPPORTED is False
+                and expression.find_ancestor(exp.AggFunc)
```

Comment:
1. You can do `expression.find_ancestor(exp.AggFunc, exp.Select) is exp.AggFunc` to avoid walking up the whole AST (i.e. a perf. improvement)
2. What if there's no `AggFunc` but this is used in the context of an `ORDER BY`? Is this handled in the following `elif` branch or something?

Reply:
Re (2): This is why BQ is `False` instead of `None` afaict, because in the context of a simple [ORDER BY](https://cloud.google.com/bigquery/docs/reference/standard-sql/query-syntax#order_by_clause) we can use both sort & null order. However, that is not the case in window specs & aggregate functions it seems.